client/src/app/create/create.component.ts

```diff
@@ -0,0 +1,55 @@
+import {Component, OnInit} from '@angular/core';
+import {EventEmitter} from "@angular/common/src/facade/async";
+import {FormGroup, FormBuilder, Validators} from "@angular/forms";
+
+interface Word{
```

Comment:
Interface możesz pobrać z komponentu learn, bo zaimplementowałem identyczny. I proponuje obiekt interfaceu nazywać z dużą literą "i" na początku. Ja w swoim komponencie nazwałem IWord. To też estetyczna zmiana